DP/04_06_fibo.py

```
@@ -0,0 +1,9 @@
+input = 20
+
+#1. Fibo(N) = Fibo(N-1) + Fibo(N-2)
+def fibo_recursion(n):
```

Ruff (0.13.3)

Unused function argument: `n`

(ARG001)